### PR TITLE
Test that radio button group and usemap name matching is case-sensitive

### PR DESCRIPTION
**Pull request by domenic**
This follows the spec change proposed at https://github.com/whatwg/html/pull/1941.

### CODE REVIEW

**Comment by annevk** [comment]
I don't understand the results from @wpt-stability-bot. @jgraham?

**Comment by annevk** [comment]
Tests look good. Marking "do not merge yet" to land them together with the specification change once approved.

**Comment by domenic** [comment]
I force-pushed a few times, so it got confused. I guess we should not do that in this new world.

File: html/semantics/embedded-content/the-object-element/usemap-casing.html
Comment (zcorpan) [inline html/semantics/embedded-content/the-object-element/usemap-casing.html]: Can you add a test with case-sensitive match, as a sanity test (with the opposite pass condition)? And similarly for img and radio button tests.

(It appears that Safari passes all the `object` tests but if I add another with a case-sensitive match, it still returns the `object` element rather than the `area`.)

**Review by zcorpan — CHANGES_REQUESTED** [review]

File: html/semantics/embedded-content/the-object-element/usemap-casing.html
Comment (domenic) [inline html/semantics/embedded-content/the-object-element/usemap-casing.html]: Interesting. Some, but not all, browsers seem to fail such a sanity check for object elements. I also am getting different results between the wpt-stability-bot (presumably Linux) and my Windows 10 computer.

I guess it'll be a good test to add regardless.

**Review by zcorpan — APPROVED** [review]

**Comment by zcorpan** [comment]
Thanks. The sanityCheck test fails in Firefox because of https://bugzilla.mozilla.org/show_bug.cgi?id=1227469

**Pull request merged**
